slsDetectorSoftware/src/CmdProxy.cpp

```diff
@@ -991,6 +991,43 @@ std::string CmdProxy::DetectorStatus(int action) {
     return os.str();
 }
 
+std::string CmdProxy::Scan(int action) {
+    std::ostringstream os;
+    os << cmd << ' ';
+    if (action == defs::HELP_ACTION) {
+        os << "[dac_name|0|trimbit_scan] [start_val] [stop_val] "
+              "[step_size]\n\tConfigures to scan "
+              "dac and sets number of frames to number of steps. Must acquire "
+              "after this. To cancel the scan configuration "
+              "set dac to '0' without further arguments. This also sets number "
+              "of frames back to 1."
+              "\n\t[Eiger]Use trimbit_scan as dac name for a trimbit scan."
+           << '\n';
+    } else if (action == defs::GET_ACTION) {
+        if (args.size() != 0) {
+            WrongNumberOfParameters(0);
+        }
+        auto t = det->getScan();
```

Comment:
Would it be possible to report back the scan that is configured?

Reply:
This is something I wanted to talk about. Maybe we can discuss this over code share or a call

Reply:
Would be something like 
`
#command line
./g scan
scan vrf 0 2000 20

//API
ScanParameters Detector::getScan() //or a smarter name

struct ScaParameters{
   dacIndex dac;
   int start;
   int stop;
   int step
};

 `

Reply:
perfect. And as discussed over the phone, Ill add the dacSettleTime as well